src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
I have to remove margin-bottom, but dont know where should i incert it? I think, its not necessary to use mixin here